src/Test/Unit/phpunit.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
+<phpunit
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="../../../vendor/phpunit/phpunit/phpunit.xsd"
+    beStrictAboutChangesToGlobalState="true"
+    beStrictAboutCoverageMetadata="true"
+    beStrictAboutOutputDuringTests="true"
+    beStrictAboutTestsThatDoNotTestAnything="true"
+    bootstrap="../../../vendor/autoload.php"
+    cacheDirectory="../../../.build/phpunit/unit/"
+    cacheResult="true"
+    colors="true"
+    columns="max"
+    displayDetailsOnIncompleteTests="true"
+    displayDetailsOnSkippedTests="true"
+    displayDetailsOnTestsThatTriggerDeprecations="true"
+    displayDetailsOnTestsThatTriggerErrors="true"
+    displayDetailsOnTestsThatTriggerNotices="true"
+    displayDetailsOnTestsThatTriggerWarnings="true"
+    executionOrder="random"
+    requireCoverageMetadata="true"
+    stopOnError="false"
+    stopOnFailure="false"
+    stopOnIncomplete="false"
+    stopOnSkipped="false"
+>
+    <coverage includeUncoveredFiles="true">
+        <exclude>
+            <directory>.</directory>
+            <directory>../Performance/</directory>
+            <directory>../Util/</directory>
+        </exclude>
```

Comment:
When you declare test code in subdirectories of production code with separate configuration files, managing exclusions for collecting code coverage with `phpunit/phpunit` in `phpunit.xml` is a lot of work.